### PR TITLE
survey proprietary reset form behaviour

### PR DESCRIPTION
**Pull request by anissa-agahchen**

### CODE REVIEW

File: app/src/features/surveys/view/components/SurveyProprietaryData.tsx [hunk 1]
```diff
@@ -105,39 +105,22 @@ const SurveyProprietaryData: React.FC<ISurveyProprietaryDataProps> = (props) =>
     }
 
     setOpenEditDialog(true);
+    console.log(surveyProprietorFormData);
```
Comment (sdevalapurkar) [inline app/src/features/surveys/view/components/SurveyProprietaryData.tsx]: Remove log 🙂

**Review by sdevalapurkar — DISMISSED** [review]
Lgtm, small comment

File: app/src/features/surveys/view/components/SurveyProprietaryData.tsx (hunk 1 above)
Comment (anissa-agahchen) [reply app/src/features/surveys/view/components/SurveyProprietaryData.tsx]: thanks!  good catch

File: app/src/features/surveys/view/components/SurveyProprietaryData.tsx [hunk 2]
```diff
@@ -68,83 +68,67 @@ const SurveyProprietaryData: React.FC<ISurveyProprietaryDataProps> = (props) =>
...
-
-        if (!response) {
-          showErrorDialog({ open: true });
-          return;
-        }
-      } catch (error) {
-        const apiError = error as APIError;
-        showErrorDialog({ dialogText: apiError.message, open: true });
+    if (!survey_proprietor) {
+      setSurveyProprietorFormData(ProprietaryDataInitialValues);
```
Comment (NickPhura) [inline app/src/features/surveys/view/components/SurveyProprietaryData.tsx:72]:
```suggestion
      setSurveyProprietorFormData(ProprietaryDataInitialValues);
      setOpenEditDialog(true);
```

**Review by NickPhura — APPROVED** [review]

**Pull request merged**